application/src/main/kotlin/dev/hypest/pis/restaurants/adapter/in/rest/RestaurantsController.kt

```diff
@@ -0,0 +1,90 @@
+@file:Suppress("InvalidPackageDeclaration")
+
+package dev.hypest.pis.restaurants.adapter.`in`.rest
+
+import dev.hypest.pis.common.UuidWrapper
+import dev.hypest.pis.restaurants.adapter.`in`.mapper.ProductMapper.mapToCreateProductCommand
+import dev.hypest.pis.restaurants.adapter.`in`.mapper.ProductMapper.mapToUpdateProductCommand
+import dev.hypest.pis.restaurants.adapter.`in`.mapper.RestaurantMapper.mapToCreateRestaurantCommand
+import dev.hypest.pis.restaurants.adapter.`in`.mapper.RestaurantMapper.mapToUpdateRestaurantCommand
+import dev.hypest.pis.restaurants.adapter.`in`.query.ProductQuery
+import dev.hypest.pis.restaurants.adapter.`in`.query.RestaurantQuery
+import dev.hypest.pis.restaurants.domain.products.CreateProductHandler
+import dev.hypest.pis.restaurants.domain.products.DeleteProductHandler
+import dev.hypest.pis.restaurants.domain.products.UpdateProductHandler
+import dev.hypest.pis.restaurants.domain.restaurants.CreateRestaurantHandler
+import dev.hypest.pis.restaurants.domain.restaurants.DeleteRestaurantHandler
+import dev.hypest.pis.restaurants.domain.restaurants.UpdateRestaurantHandler
+import dev.hypest.pis.restaurants.product.CreateProductRequest
+import dev.hypest.pis.restaurants.product.ProductResponse
+import dev.hypest.pis.restaurants.product.UpdateProductRequest
+import dev.hypest.pis.restaurants.restaurants.CreateRestaurantRequest
+import dev.hypest.pis.restaurants.restaurants.RestaurantResponse
+import dev.hypest.pis.restaurants.restaurants.RestaurantsApi
+import dev.hypest.pis.restaurants.restaurants.UpdateRestaurantRequest
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.annotation.Controller
+import java.util.UUID
+
+@Suppress("LongParameterList")
+@Controller("/restaurants")
+class RestaurantsController(
+    private val createRestaurantHandler: CreateRestaurantHandler,
+    private val updateRestaurantHandler: UpdateRestaurantHandler,
+    private val deleteRestaurantHandler: DeleteRestaurantHandler,
+    private val createProductHandler: CreateProductHandler,
+    private val updateProductHandler: UpdateProductHandler,
+    private val deleteProductHandler: DeleteProductHandler,
+    private val restaurantQuery: RestaurantQuery,
+    private val productQuery: ProductQuery
+) : RestaurantsApi {
+    override fun getAllRestaurants(): HttpResponse<List<RestaurantResponse>> =
```

Comment:
A pododawałbyś adnotacje tak jak masz w Api, bo patrząc na controller to nie wiadomo co sie dzieje i trzeba zerkać w 2 miejsca